<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<node pkg="first_controller" type="pid_controller.py" name="pid_controller">
	<param name="steer_control_topic"                  value="/lli/ctrl_request"/>
	<param name="car_pose_topic"                       value="/SVEA2/pose"/>
	<param name="path_topic"                           value="/SVEA2/path"/>
	<param name="waypoint_topic"                       value="/leader/pose"/> 
	<param name="command_controller_topic"             value="/start_stop_controller"/>
	<param name="/command_controller_follow_topic"     value="/start_stop_follow"/>
	</node>
</launch>


